cmd/demo/node.go

```diff
@@ -443,14 +462,21 @@ func (n *node) Info(args []string) error {
 	n.log.Traceln("Info...")
 
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', tabwriter.Debug)
-	fmt.Fprintf(w, "Peer\tPhase\tVersion\tMy Ξ\tPeer Ξ\t\n")
+	fmt.Fprintf(w, "Peer\tPhase\tVersion\tMy Ξ\tPeer Ξ\tMy OnChain Ξ\tPeer OnChain Ξ\t\n")
 	for alias, peer := range n.peers {
+		onChainBalListWei, err := getOnChainBal([]wallet.Address{n.onChain.Address(), peer.perunID})
+		if err != nil {
+			return errors.WithStack(err)
```

Comment:
Just `err` or `errors.WithMessage…`.

Reply:
Addressed in this commit - 85c0766. Added `errors.Wrap..` in `getOnChainBal` definition and used just `err` here.